common/dna_data/wc_human_dna.txt

```diff
@@ -10520,6 +10520,126 @@ godfrey_dna = { # by runtd
 	enabled=yes
 }
 
+natalie_seline_dna = { 
+	portrait_info = { 
```

Comment:
her dna gives bunch of errors. You didnt check `error.log`. You can ask in dev chat how to properly make dna.

Reply:
i did and they said i could just paste into the mod files. i'll remove for now until i have a better understanding as this is not being used for the feature.